glpi/files/etc/php/conf.d/glpi.ini

```diff
@@ -5,3 +5,12 @@ session.cookie_samesite = "Strict"
 
 ; Do not expose PHP version
 expose_php = off
+
+; allow posting larger files (e.g., for attachments)
+post_max_size = 20M
+upload_max_filesize = 20M
```

Comment:
The GLPI upload handler split files into chunks to be able to upload large files without having to change this.